### DISCUSSION TITLE
Fix RTL language text layout, When?

### DISCUSSION THREAD

**Discussion by FerasK0** [robertodoering/harpy#656] [General]
Hi!
It has been 4 months, is this fix coming to 0.9.0 or to the one after it?
because it's still present in 0.9.0 and although it's marked as easy it hasn't been done yet :/
![image](https://user-images.githubusercontent.com/83197825/169270657-2507a110-a88d-4252-bccd-8c1cba678cff.png)

**Comment by FerasK0** [comment]
Finally!
Just tried 0.9.3 and Arabic works flawlessly, except when making a tweet the problem still exists, see pictures below.
FYI it's the same text just copied and pasted.

The text:
انا اتحدث العربية i speak English واحد اثنان ثلاثة

# **Harpy**
![Screenshot_20220606-091106_harpy free.png](https://user-images.githubusercontent.com/83197825/172105354-b58e500a-8b55-4fa9-b763-b88473e5e6e1.png)

# **Twitter**
![Screenshot_20220606-091024_Twitter.png](https://user-images.githubusercontent.com/83197825/172105426-9af2cdc3-5489-455c-b39d-bb0868311a87.png)

Twitter looks more natural when typing two languages, please consider :)

**Reply by robertodoering** [reply]
Thanks @FerasK0! I'm glad you like the improved right-to-left support :)
I'll create a new ticket for better support when composing a Tweet.

If you find any other issues with the layout please let me know as I'm not very familiar with right-to-left languages and their layout changes :)

**Reply by FerasK0** [reply]
Will do sir, and I'll make sure to widen the Arabic users to insure that it's well supported all around. ^_^
